From c18158e332d9bb174ca0f1b600fe10418aceaaad Mon Sep 17 00:00:00 2001
From: Kevin Zhang <akz002@freescale.com>
Date: Tue, 22 Jan 2008 15:30:17 -0600
Subject: [PATCH] ENGR00062277 fix build break after adding mx37 nand

Description:
Fix build break on i.MX31 3stack after adding NAND for i.MX37 3stack
board.
---
 drivers/mtd/nand/mxc_nd2.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/drivers/mtd/nand/mxc_nd2.h b/drivers/mtd/nand/mxc_nd2.h
index 1483ae4..5372d17 100644
--- a/drivers/mtd/nand/mxc_nd2.h
+++ b/drivers/mtd/nand/mxc_nd2.h
@@ -294,6 +294,11 @@
 #define SPARE_COUNT       4
 #define SPARE_SIZE        (SPARE_LEN * SPARE_COUNT)
 
+/* read column 464-465 byte but only 464 for bad block marker */
+#define BAD_BLK_MARKER_MA	IO_ADDRESS(NFC_BASE_ADDR + 0x600 + 464)
+/* read column 0-1 byte, but only 1 is used for swapped main area data */
+#define BAD_BLK_MARKER_SP	IO_ADDRESS(NFC_BASE_ADDR + 0x800)
+
 #define IS_4BIT_ECC       1
 #define SET_SPAS(v)
 #define SET_ECC_MODE(v)
-- 
1.5.4.4

